worker/src/handlers/schedule.ts

```diff
@@ -96,7 +96,12 @@ async function processQueue(sentry: Toucan): Promise<void> {
     core_analytics[timestampString] = queue_data;
 
     sentry.addBreadcrumb({ message: "Trigger Netlify build" });
-    const resp = await fetch(NETLIFY_BUILD_HOOK, { method: "POST" });
+    const resp = await fetch(
+      WORKER_ENV === "production"
+        ? NETLIFY_BUILD_HOOK
+        : NETLIFY_BUILD_HOOK_STAGING,
```

Comment:
We should always read from the same environment variables. The env variables should have different values based on their environment.

Reply:
can they have that?

Reply:
This is for the worker... yes, they can have same name